interactions/api/events/discord.py

```diff
@@ -1,21 +1,6 @@
 """
 These are events dispatched by Discord. This is intended as a reference so you know what data to expect for each event.
 
-??? Hint "Example Usage:"
-    The event classes outlined here are in `CamelCase` to comply with Class naming convention, however the event names
-    are actually in `lower_case_with_underscores` so your listeners should be named as following:
-
-    ```python
-    @listen()
-    def on_ready():
-        # ready events pass no data, so dont have params
-        print("Im ready!")
-
-    @listen()
-    def on_guild_join(event):
-        # guild_create events pass a guild object, expect a single param
-        print(f"{event.guild.name} created")
-    ```
```

Comment:
Why remove this? It's still useful - instead why not just link to the guide in addition?

Reply:
I think we really should leave at least a small explanation on usage here

Reply:
...why? The guide covers that well, and it's not like many people who look at the API reference are going to be unwilling to look at the guides or something.

Reply:
The objective of documentation should be to provide the information with minimum friction. At a glance documentation is fantastic for that. 

All removing an example on this page achieves is increasing friction for getting information for very little benefit. All I'm asking is for the basic usage example to stay - the guide should remain the primary source

Reply:
I'll agree to disagree on a number of your points - I'll add a basic example back in, though I'll be suggesting to use the event object instead of the name.